raiden/raiden_event_handler.py

```diff
@@ -210,7 +224,7 @@ def handle_paymentsentsuccess(
         # With the introduction of the lock we should always get
         # here only once per identifier so payment_status should always exist
         # see: https://github.com/raiden-network/raiden/pull/3191
-        payment_status.payment_done.set(True)
+        payment_status.payment_done.set(payment_sent_success_event.secret)
```

Comment:
made comments about this in https://github.com/raiden-network/raiden/pull/3560

Reply:
Noted